app/controllers/calls_controller.rb

```diff
@@ -2,32 +2,39 @@ class CallsController < ApplicationController
 
   after_filter :set_header
 
+
+
   # Public: initiates the call process via a request from twillio
   #
   # CallSid - default param from twilio (required)
   def start
     response = Twilio::TwiML::Response.new do |r|
-      r.Say 'We need you to connect with your congressperson and senator. We are going to put you in touch with '
+      r.Play AudioFileFetcher.file_for_key('intro_message')
       r.Redirect calls_new_connection_url, method: 'get'
     end
 
     render_twiml response
   end
 
-  # Public: initiates a new outgoing call to congress
+  # Public: initiates a new outgoing call to congress. If ENV FAKE_CONGRESS_NUMBER is
+  # set, then the all calls will be routed to that number.
   #
   # CallSid - default param from twilio (required)
   def new_connection
     response = Twilio::TwiML::Response.new do |r|
-      if active_call.target_legislators.any?
+      if active_call.next_target.present?
         connection = active_call.create_connection!
-        r.Say 'We will connect you in just a moment. Press star at any time to disconnect from your legislator.'
+        # r.Play AudioFileFetcher.file_for_key('connecting_local')
+        r.Play AudioFileFetcher.file_for_key('connecting_to_representative')
+        r.Play AudioFileFetcher.file_for_key('star_to_disconnect')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.